packages/next/client/image.tsx

```diff
@@ -318,6 +360,13 @@ export default function Image({
 
     objectFit,
     objectPosition,
+
+    ...(shouldShowBlurryPlaceholder
+      ? {
+          backgroundSize: 'cover',
+          backgroundImage: `url("${blurDataURL}")`,
```

Comment:
wouldn't this be possible as a possible default? @Joonpark13 
```suggestion
          backgroundImage: `url("${blurDataURL ?? `/_next/image?url=${src}&w=16&q=75`}")`,
```